pennylane/gradients/parameter_shift.py

```diff
@@ -1489,6 +1486,31 @@ def processing_fn(results):
                 combined_grad.append(meas_grad)
             return tuple(combined_grad)
 
+        # If there are unsupported parameters, we must process
+        # the quantum results separately, once for the fallback
+        # function and once for the parameter-shift rule, and recombine.
+        def processing_fn(results):
+            unsupported_grads = results[:fallback_len]
+            supported_grads = results[fallback_len:]

```

Comment:
Minor: these names are misleading since they don't refer to gradients. Maybe `unsupported_res` and `supported_res`?

Reply:
Sg. :+1: Renamed them.